spec/javascript/components/atoms/Column.test.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import { shallow } from 'enzyme'
+import Column from '~/components/atoms/Column'
+
+describe('Column', () => {
+  test('it should render a column with the correct properties', () => {
+    const wrapper = shallow(
+      <Column span='span-test' end form>
+        <div className='some-div'></div>
```

Comment:
super minor nit: components with no children should use the closing shorthand like `<div className='some-div' />`

I should probably add a linting rule for this